gnupg-pkcs11-scd/command.c

```diff
@@ -1500,7 +1607,7 @@ gpg_error_t cmd_pkdecrypt (assuan_context_t ctx, char *line)
 		(error = common_map_pkcs11_error (
 			pkcs11h_certificate_decryptAny (
 				cert,
-				CKM_RSA_PKCS, 
+				CKM_RSA_PKCS,
```

Comment:
also cleanup should go in own pull request so we can merge them regardless and simplify this patch

Reply:
I've re-added this whitespace error.